rotkehlchen/db/upgrades/v47_v48.py

```diff
@@ -147,7 +149,7 @@ def _convert_trades_to_swap_events(write_cursor: 'DBCursor') -> None:
             'INSERT OR IGNORE INTO history_events(entry_type, event_identifier, '
             'sequence_index, timestamp, location, location_label, asset, amount, '
             'notes, type, subtype, extra_data) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)',
-            (event.serialize_for_db()[0][2] for event in new_events),
+            [event._serialize_base_tuple_for_db()[2] for event in new_events],
```

Comment:
using the underlying _serialize, which is essentially the same for SwapEvent (for now) seems like a good reason to improve the speed by saving 1 function call per iteration.